lua/catppuccin/groups/integrations/treesitter.lua

```diff
@@ -12,122 +12,131 @@ If you want to stay on nvim 0.7, either disable the integration or pin catppucci
 
 		-- Keywords
 		["@keyword"] = { link = "Keyword" }, -- For keywords that don't fall in previous categories.
 		["@keyword.function"] = { fg = C.mauve, style = O.styles.keywords or {} }, -- For keywords used to define a function.
 		["@keyword.operator"] = { fg = C.mauve, style = O.styles.operators or {} }, -- For new keyword operator
+		["@keyword.import"] = { link = "Include" }, -- For includes: #include in C, use or extern crate in Rust, or require in Lua.
+		["@keyword.storage"] = { link = "StorageClass" }, -- visibility/life-time/etc. modifiers (e.g. `static`)
+		["@keyword.repeat"] = { link = "Repeat" }, -- For keywords related to loops.
 		["@keyword.return"] = { fg = C.mauve, style = O.styles.keywords or {} },
+		["@keyword.exception"] = { link = "Exception" }, -- For exception related keywords.
+
+		["@keyword.conditional"] = { link = "Conditional" }, -- For keywords related to conditionnals.
+
+		["@keyword.directive"] = { link = "PreProc" }, -- various preprocessor directives & shebangs
+		["@keyword.directive.define"] = { link = "Define" }, -- preprocessor definition directives
 		-- JS & derivative
 		["@keyword.export"] = { fg = C.sky, style = O.styles.keywords },
 
-		["@conditional"] = { link = "Conditional" }, -- For keywords related to conditionnals.
-		["@repeat"] = { link = "Repeat" }, -- For keywords related to loops.
-		-- @debug            ; keywords related to debugging
-		["@label"] = { link = "Label" }, -- For labels: label: in C and :label: in Lua.
-		["@include"] = { link = "Include" }, -- For includes: #include in C, use or extern crate in Rust, or require in Lua.
-		["@exception"] = { link = "Exception" }, -- For exception related keywords.
+		-- Punctuation
+		["@punctuation.delimiter"] = { link = "Delimiter" }, -- For delimiters (e.g. `;` / `.` / `,`).
+		["@punctuation.bracket"] = { fg = C.overlay2 }, -- For brackets and parenthesis.
+		["@punctuation.special"] = { link = "Special" }, -- For special punctuation that does not fall in the categories before (e.g. `{}` in string interpolation).
 
-		-- Types
-		["@type"] = { link = "Type" }, -- For types.
-		["@type.builtin"] = { fg = C.yellow, style = O.styles.properties or { "italic" } }, -- For builtin types.
-		["@type.definition"] = { link = "Type" }, -- type definitions (e.g. `typedef` in C)
-		["@type.qualifier"] = { link = "Keyword" }, -- type qualifiers (e.g. `const`)
+		-- Comment
+		["@comment"] = { link = "Comment" },
 
-		["@storageclass"] = { link = "StorageClass" }, -- visibility/life-time/etc. modifiers (e.g. `static`)
-		["@attribute"] = { link = "Constant" }, -- attribute annotations (e.g. Python decorators)
-		["@field"] = { fg = C.lavender }, -- For fields.
-		["@property"] = { fg = C.lavender, style = O.styles.properties or {} }, -- Same as TSField.
+		["comment.error"] = { fg = C.base, bg = C.red },
+		["comment.warning"] = { fg = C.base, bg = C.yellow },
+		["comment.note"] = { fg = C.base, bg = C.blue },
```

Comment:
missing `comment.todo`. Also they should be `@comment.*`

Reply:
The former `@text.todo` has two different meanings, 
- one for todo in comments, now `@comment.warning`
- one for todo lists, now `@markup.list.checked`/`@markup.list.unchecked`

So the `comment.todo` is a new capture, I think it's better not to add new highlights for now, especially since this highlight hasn't been utilized.(it's not exists in nvim-treesitter now)

Reply:
Continue on https://github.com/catppuccin/nvim/pull/645